django_cloud_deploy/cli/cloudify.py

```diff
@@ -37,10 +37,21 @@ def add_arguments(parser):
 
     parser.add_argument(
         '--project-path',
-        dest='django_directory_path',
+        dest='django_directory_path_cloudify',
         help='The location where the generated Django project code should be '
         'stored.')
 
+    parser.add_argument(
+        '--django_requirements_path',
+        dest='django_requirements_path',
+        help='The location of the Django project requirements.txt.')
+
+    parser.add_argument(
```

Comment:
those were during testing i wanted to add since they were missing. I do think they could of been a seperate pr but since this one was still relatively small i thoght there wouldnt be a problem adding it directly.

Reply:
There should also be '--django_settings_path'.

Reply:
yeah, i think that got added in between this pr :) will add it

Reply:
Sure.